2022-02-18 16:11:31,097 INFO    MainThread:13207 [wandb_setup.py:_flush():71] setting env: {}
2022-02-18 16:11:31,097 INFO    MainThread:13207 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-18 16:11:31,097 INFO    MainThread:13207 [wandb_init.py:_log_setup():371] Logging user logs to /home/zinzinbin/codes_for_study/gnn/MNIST_GNN/wandb/run-20220218_161131-2p5x36qq/logs/debug.log
2022-02-18 16:11:31,097 INFO    MainThread:13207 [wandb_init.py:_log_setup():372] Logging internal logs to /home/zinzinbin/codes_for_study/gnn/MNIST_GNN/wandb/run-20220218_161131-2p5x36qq/logs/debug-internal.log
2022-02-18 16:11:31,097 INFO    MainThread:13207 [wandb_init.py:init():404] calling init triggers
2022-02-18 16:11:31,097 INFO    MainThread:13207 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-18 16:11:31,097 INFO    MainThread:13207 [wandb_init.py:init():460] starting backend
2022-02-18 16:11:31,098 INFO    MainThread:13207 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-18 16:11:31,103 INFO    MainThread:13207 [backend.py:ensure_launched():216] starting backend process...
2022-02-18 16:11:31,106 INFO    MainThread:13207 [backend.py:ensure_launched():221] started backend process with pid: 13252
2022-02-18 16:11:31,106 INFO    MainThread:13207 [wandb_init.py:init():469] backend started and connected
2022-02-18 16:11:31,108 INFO    MainThread:13207 [wandb_init.py:init():533] updated telemetry
2022-02-18 16:11:31,116 INFO    MainThread:13207 [wandb_init.py:init():563] communicating current version
2022-02-18 16:11:31,560 INFO    MainThread:13207 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-18 16:11:31,560 INFO    MainThread:13207 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-18 16:11:31,862 INFO    MainThread:13207 [wandb_init.py:init():606] starting run threads in backend
2022-02-18 16:11:32,953 INFO    MainThread:13207 [wandb_run.py:_console_start():1810] atexit reg
2022-02-18 16:11:32,953 INFO    MainThread:13207 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-18 16:11:32,953 INFO    MainThread:13207 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-18 16:11:32,954 INFO    MainThread:13207 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-18 16:11:32,954 INFO    MainThread:13207 [wandb_init.py:init():633] run started, returning control to user process
2022-02-18 16:11:45,508 INFO    MainThread:13207 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-18 16:11:45,509 INFO    MainThread:13207 [wandb_run.py:_restore():1752] restore
2022-02-18 16:11:48,168 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-02-18 16:11:48,278 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-02-18 16:11:48,512 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-02-18 16:11:48,874 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 16579
}

2022-02-18 16:11:48,975 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 16579
}

2022-02-18 16:11:49,078 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 16579
}

2022-02-18 16:11:49,179 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}

2022-02-18 16:11:49,280 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}

2022-02-18 16:11:49,381 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}

2022-02-18 16:11:49,482 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}

2022-02-18 16:11:49,583 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}

2022-02-18 16:11:49,684 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}

2022-02-18 16:11:50,020 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}

2022-02-18 16:11:50,515 INFO    MainThread:13207 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16579
  total_bytes: 16579
}
local_info {
}

2022-02-18 16:11:51,652 INFO    MainThread:13207 [wandb_run.py:_append_files():2180] logging synced files
